modules/govuk/manifests/apps/ckan/cronjobs.pp

```diff
@@ -59,15 +59,15 @@
   }
 
   govuk::apps::ckan::ckan_cronjob { 'Environment Agency harvester stop existing processes':
-    ensure       => $ensure,
+    ensure       => 'absent',
     ckan_command => 'harvester job-abort environment-agency-data-sharing-platform',
     weekday      => '5',
     hour         => '15',
     minute       => '55',
   }
 
   govuk::apps::ckan::ckan_cronjob { 'Environment Agency harvester run':
-    ensure       => $ensure,
+    ensure       => 'absent',
```

Comment:
I don't have context on this so I'm just double-checking that this commit is here intentionally, since you didn't mention it in the PR description.